Interfaces/API/Dirac.py

```diff
@@ -1261,6 +1261,22 @@ def addFile( self, lfn, fullPath, diracSE, fileGuid = None, printOutput = False
     return result
 
   #############################################################################
+  def getFilesFromDirectory( self, directory, destdir, wildcard='*', days=0):
```

Comment:
I see, maybe it is because of this https://its.cern.ch/jira/browse/LHCBDIRAC-322?